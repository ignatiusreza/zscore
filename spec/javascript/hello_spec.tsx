import React from 'react';
import Hello from 'hello';

describe('Hello', () => {
  it('works for testing react component', () => {
    expect(<Hello />).toMatchSnapshot();
  });
});
