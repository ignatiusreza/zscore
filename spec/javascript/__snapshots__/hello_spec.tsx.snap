// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hello works for testing react component 1`] = `
<div
  className="hello"
>
  Hello 
  David
  !
</div>
`;
